src/Qdrant.Client/Grpc/Conditions.cs

```diff
@@ -0,0 +1,259 @@
+namespace Qdrant.Client.Grpc;
+
+/// <summary>
+/// Methods to simplify building <see cref="Condition"/>
+/// </summary>
+public static class Conditions
```

Comment:
Good point.

I think the value of having the names match is higher than the value of having the methods on Condition as opposed to Conditions... Realistically, most users using this will likely follow our docs sample and just do a `using static` for Conditions, at which point it becomes irrelevant exactly where the methods live.

So I'd vote for leaving things like this.